config/deploy/development.rb

```diff
@@ -0,0 +1,4 @@
+set :host, Figs.env.development["host"]
```

Comment:
I think what you can do is:

```ruby
set :host, { ENV["host"] }
```

What that will do is lazy load the `:host` when it needs it, and at that point (after figs has loaded) it will be in the right environment.